### PR TITLE
Add var-name sub linter in revive

### PR DESCRIPTION
**Pull request by congqixia**
See also #8201
/kind improvement

### CODE REVIEW

File: internal/querycoordv2/checkers/controller.go [hunk 1]
```diff
@@ -34,11 +34,11 @@ import (
...
 var (
-	Segment_Checker = "segment_checker"
-	Channel_Checker = "channel_checker"
-	Balance_Checker = "balance_checker"
-	Index_Checker   = "index_checker"
-	CheckerOrder    = []string{Channel_Checker, Segment_Checker, Balance_Checker, Index_Checker}
+	segmentChecker = "segment_checker"
+	channelChecker = "channel_checker"
+	balanceChecker = "balance_checker"
+	indexChecker   = "index_checker"
```
Comment (yah01) [inline internal/querycoordv2/checkers/controller.go:40]: these should be const

**Review by yah01 — CHANGES_REQUESTED** [review]

File: internal/querycoordv2/checkers/controller.go (hunk 1 above)
Comment (congqixia) [reply internal/querycoordv2/checkers/controller.go:38]: done

**Review by yah01 — COMMENTED** [review]
/lgtm

**Pull request merged**